proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,84 @@ The Working Group identified four possible workflows for event passing, based on
+Having listeners return evolvable events also posed a challenge.  That pattern is not used by any known implementations in PHP or elsewhere.  It also relies on the listener to remember to return the Event (extra work for the listener author) and to not return some other, new object that might not be fully compatible with later listeners (such as a subclass or superclass of the Event).
 
 Immutable events also rely on the event definer to respect the admonition to be immutable.  Events are, by nature, very loosely designed and the potential for implementers to ignore that part of the spec, even inadvertently, is high.
 
 That left two possible options:
 
-* Allow tasks to be mutable.
+* Allow events to be mutable.
 * Require, but be unable to enforce, immutable objects with a high-ceremony interface, more work for listener authors, and a higher potential for breakage that may not be detectable at compile time.
 
-Given those options the Working Group felt mutable tasks were the safer alternative.
+Given those options the Working Group felt mutable events were the safer alternative.
 
-The Message Notification use case, however, has no need for return communication so those issues are moot.  Those objects therefore MUST be treated as immutable, regardless of what the object's methods are.
+That said, there is no requirement that an Event be mutable.  Implementers should provide mutator methods on an Event object if and only if it is necessary and appropriate to the use case at hand.
 
 ### 4.4 Listener registration
 
-Experimentation during development of the specification determined that there were a wide range of viable, legitimate means by which a Notifier or Processor could be informed of a listener.
+Experimentation during development of the specification determined that there were a wide range of viable, legitimate means by which a Dispatcher could be informed of a listener.  A listener:
 
-* A listener could be registered explicitly; it could be the registered explicitly based on reflection of its signature;
+* could be registered explicitly;
+* it could be the registered explicitly based on reflection of its signature;
 * it could be registered with a numeric priority order;
 * it could be registered using a before/after mechanism to control ordering more precisely;
 * it could be registered from a service container;
 * it could use a pre-compile step to generate code;
-* it could be based on method names on objects in the event itself.
+* it could be based on method names on objects in the event itself;
+* it could be limited to certain situations or contexts based on arbitrarily complex logic (only for certain users, only on certain days, only if certain system settings are present, etc).
```

Comment:
The subject of each of these bullet points is "A listener"; remove the "it" pre-amble to each, as is done correctly in the first point.